modules/dream2nix/rust-crane/interface.nix

```diff
@@ -65,6 +65,11 @@ in {
       description = "Flags to add when testing the crate";
       default = [];
     };
+    testFlagsExtra = {
+      type = t.str;
+      description = "additional flags to be passed in the cargoTestCommand invocation";
+      default = "--no-run";
```

Comment:
TODO:FIXME: this default should not be streamlined. 

crane expect a string here. Without default everyone needs to set it. With some default some crane default would be overwritten. Make this interface optional.